src/main/java/com/pretz/dyntreeserver/service/exceptions/AuthExceptionHandler.java

```diff
@@ -12,12 +12,12 @@
 
     @ExceptionHandler(value = AuthException.class)
     protected ResponseEntity<String> handleAuthException(AuthException aEx) {
-        return new ResponseEntity<>(aEx.authExceptionMsg + aEx.userName, new HttpHeaders(), HttpStatus.UNAUTHORIZED);
+        return new ResponseEntity<>(AuthException.authExceptionMsg + aEx.userName, new HttpHeaders(), HttpStatus.UNAUTHORIZED);
```

Comment:
Maybe instead of accessing static properties and concatenating two internal values here, add a `getMessage` method to `AuthException`?

Reply:
[Fixed here] (https://github.com/Elm0xz/dyn-tree-server/commit/f77a8f1419de9a951710ecd587612e0b5315c75c). Actually I've solved this small issue in another way - I'm passing concatenated String by Exception classes field. Unfortunately, in Java super constructor has to be in first line when called by extending class, so the result is still a bit off.